src/espIdf/openOcd/openOcdManager.ts

```diff
@@ -222,6 +234,19 @@ export class OpenOCDManager extends EventEmitter {
       this.encounteredErrors = true;
       data = typeof data === "string" ? Buffer.from(data) : data;
       this.sendToOutputChannel(data);
+      
+      // Parse adapter serial number from log output
+      const serialNumber = parseAdapterSerialFromLog(data);
+      if (serialNumber && this.workspace) {
+        storeAdapterSerial(this.workspace, serialNumber).catch((error) => {
+          Logger.error(
+            "Failed to store adapter serial number",
+            error,
+            "OpenOCDManager stderr"
```

Comment:
Better to improve the error tag here instead of `OpenOCDManager stderr` to `storeAdapterSerial`